client/signup/steps/plans/index.jsx

```diff
@@ -101,13 +102,23 @@ module.exports = React.createClass( {
 
 	plansSelection: function() {
 		let headerText = this.translate( 'Pick a plan that\'s right for you.' ),
+			subHeaderText;
+
+		if ( config.isEnabled( 'upgrades/free-trials' ) ) {
+			subHeaderText = this.translate(
+				'Try WordPress.com Premium or Business free for 14 days, no credit card required.'
+			);
+		} else {
 			subHeaderText = this.translate(
 				'Not sure which plan to choose? Take a look at our {{a}}plan comparison chart{{/a}}.',
-				{ components: { a: <a
-					href={ this.comparePlansUrl() }
-					onClick={ this.handleComparePlansLinkClick.bind( null, 'header' ) } /> } }
+				{ 
+					components: { a: <a
+						href={ this.comparePlansUrl() }
+						onClick={ this.handleComparePlansLinkClick.bind( null, 'header' ) } /> }
+				}
 			);
 
+		}
 		return (
```

Comment:
That’s coming along nicely. The only thing that I would do is add a few blank lines here and in the `main.jsx` file (above `return <div className="plans__trial-copy">...` and below `{ this.renderTrialCopy() }`). This is certainly not mandatory but that would improve readability.